pkg/utils/mainhelper/init.go

```diff
@@ -49,7 +49,7 @@ type InitRes struct {
 // it returns a result object that contains useful artifacts for later use.
 // Unlike sharedmain.Main, Init is meant to be run as a helper function in any main
 // functions, while sharedmain.Main runs controllers with predefined method signatures.
-func Init(component string, opts ...InitOption) (context.Context, *InitRes) {
```

Comment:
How about making metricNamespace an option, and by default it's the same as `component`? e.g.m Init(component, WithMetricNamespace("trigger"))

Reply:
Updated, great suggestion!